eventsource/stores/sqlstore/store_test.go

```diff
@@ -197,5 +216,5 @@ CREATE TABLE %s (
 	assert.Nil(t, err, "Could not get events")
 	assert.Equal(t, 1, len(events))
 
-	db.Exec("DROP TABLE $1", tmpTableName)
+	db.Exec(fmt.Sprintf("DROP TABLE IF EXISTS %s", tmpTableName))
```

Comment:
L178 should be `require.NoError(t, err...` 
It should not continue if the creation of the table wasn't succesful. It could be dangerous if the table that already exists got deleted